platform/security/core/security-core-impl/src/main/java/ddf/security/assertion/jwt/impl/SecurityAssertionJwt.java

```diff
@@ -59,29 +61,26 @@ public SecurityAssertionJwt(OidcProfile profile) {
   public SecurityAssertionJwt(OidcProfile profile, List<String> usernameAttributeList) {
     this.profile = profile;
     this.usernameAttributeList = usernameAttributeList;
-    try {
```

Comment:
📘 Instead of using a `JWTClaimsSet` from the `ID_TOKEN`, uses the `attributes` from the `OidcProfile` instead.

Changes were made to the `OidcProfileCreator` to make sure the `attributes` are set correctly.